politeiad/backend/gitbe/gitbe_test.go

```diff
@@ -308,10 +308,6 @@ func TestAnchorWithCommits(t *testing.T) {
 		t.Fatalf("invalid anchor type %v expected %v", anchor3.Type,
 			AnchorVerified)
 	}
-	if anchor3.Transaction != expectedTestTX {
```

Comment:
This portion of the test was removed, since we are no longer updating anchor records after confirmation.